about-page.css

```diff
@@ -0,0 +1,795 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  font-family: "Lato", "Gothic A1", sans-serif;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media only screen and (max-width: 767px) {
+  /* Ham starts */
+  .ham-popup {
+    background: #017f94;
+    position: fixed;
+    height: 100%;
+    width: 100%;
+    z-index: 99;
+    display: none;
+  }
```

Comment:
- [ ] I like that you're trying to add your own flavor 🧂 here but as part of the project requirements the pages you build should use the exact same color, for text, backgrounds, etc... So I'd kindly suggest you stick to the color swatch for designing the mobile menu here.
  - Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`